src/utils/encode.ts

```diff
@@ -94,6 +177,21 @@ function padString(str: string, length: number, left: boolean, padding = STRING_
 
 /**
  * Prepend string (default with '0')
+ *
+ * Pads a string to a certain length with a specific string.
+ * The padding can be applied only to the left of the input string.
+ *
+ * @param {string} str The string to pad.
+ * @param {number} length The target length for the padded string.
+ * @param {string} [padding='0'] The string to use for padding. Defaults to '0'.
+ * @returns {string} The padded string.
+ *
+ * @example
+ * ```typescript
+ * const myString = 'hello';
```

Comment:
This function is used for hex string, not for text. Please change to a hex num example.